client/signup/steps/domains/index.jsx

```diff
@@ -145,6 +148,33 @@ class DomainsStep extends Component {
 		return isPlansStepExistsInFutureOfFlow && ! isPlanStepSkipped;
 	};
 
+	componentWillMount() {
```

Comment:
An FYI that I received feedback on my PR [here](https://github.com/Automattic/wp-calypso/pull/59771/files#r785991500) to use `componentDidMount()` instead.